### ISSUE TITLE
Subtitles on training videos

### ISSUE THREAD

**Issue by geoffj-FUG** [FreeUKGen/FreeCENMigration#1508]
Do we need to add subtitles on the FreeCEN videos.

**Comment by AlOneill** [comment]
Yes we do. The auto-generated captions cannot cope with project-specific and some genealogical terms, for instance, and are apt to introduce further confusion or misunderstanding because there is no proper sentence structure. I have used some of them (and regularly need subtitles on YouTube videos), so this is more than guesswork.

BTW, is this story already covered elsewhere? Maybe not …

**Comment by geoffj-FUG** [comment]
Thanks Alison

 

I was at a FreeCEN Committee meeting and the question was asked. I created the story so I could follow it up afterwards. 

 

I will look to see if there is already a story. I couldn’t remember seeing one. If there is I will remove the new one.

 

Geoff
